intuita-webview/src/CreateIssue/index.tsx

```diff
@@ -14,6 +14,17 @@ import './tiptap.css';
 
 const lowlight = createLowlight(common);
 
+const convertHTMLCodeBlockToMarkdownString = (htmlSnippet: string) => {
```

Comment:
why was that needed? aren't we controlling what HTML gets created?

Reply:
The initial value we pass to tiptap editor is a HTML string; otherwise, we can't apply CSS styling.
GH doesn't handle `<pre><code class='language-typescript'>...</code></pre>`. Hence, we run this function before passing it to GH.